src/class-parsely.php

```diff
@@ -694,8 +694,14 @@ private function get_categories( int $post_id, string $delimiter = '/' ): array
 		if ( $last_tag ) {
 			$tags = explode( '/', $last_tag );
 		}
-		// remove uncategorized value from tags.
-		return array_diff( $tags, array( 'Uncategorized' ) );
+
+		// Remove default category name from tags if needed.
+		$default_category_name = get_cat_name( get_option( 'default_category' ) );
+		if ( null !== $default_category_name ) {
```

Comment:
`get_cat_name()` will never be `null`. If it fails, it will return an empty string, and that's fine to pass to `array_diff()`.

Please, just use the diff I suggested.

Reply:
I've removed the problematic `if` check.